README.md

```diff
@@ -67,10 +67,22 @@ from https://console.cloud.google.com/functions/list?authuser=0&project=dash-wal
 
 #### Trigger function
 
-Function can be triggered by publishing `dash-explore-sync-trigger` PubSub topic
+Function can be triggered by publishing `explore-dash-sync-trigger` PubSub topic
 
 ```
-gcloud pubsub topics publish dash-explore-sync-trigger
+gcloud pubsub topics publish explore-dash-sync-trigger
+```
+
+Or for the testnet mode:
+
+```
+gcloud pubsub topics publish explore-dash-sync-trigger --attribute="mode=testnet"
+```
+
+Or for the testnet mode:
+
+```
+gcloud pubsub topics publish dash-explore-sync-trigger --attribute="mode=testnet"
```

Comment:
The last line says `dash-explore-sync-trigger` -- should this be removed, or does it need other text above that says what it does?

Reply:
Yes, you are right. Removed.